autograder/cli/submission/fetch-attempts.py

```diff
@@ -0,0 +1,42 @@
+import os
+import sys
+
+import autograder.api.submission.fetchattempts
+import autograder.cli.submission.common
+
+def run(arguments):
+    result = autograder.api.submission.fetchattempts.send(arguments, exit_on_error = True)
+
+    if (not result['found-user']):
+        print("No matching user found.")
+        return 1
+
+    if (len(result['grading-results']) == 0):
+        print("No attempts found.")
+        return 2
+
+    out_dir = os.path.join(arguments.out_dir, arguments.assignment, result['grading-results'][0]['info']['user'])
```

Comment:
Instead of `arguments.assignment`, use `result['grading-results'][0]['info']['user']`.
This is because the second one is standardized (e.g. the first one could be "ho0" or "HO0").